chapter4.Rmd

```diff
@@ -214,7 +208,7 @@ When you first get a data set, you will often notice that it contains factors wi
 levels(factor_vector) <- c("name1", "name2",...)
 ```
 
-A good illustration is the raw data that is provided to you by a survey. A standard question for every questionnaire is the gender of the respondent. You remember from the previous question that this is a factor and when performing the questionnaire on the streets its levels are often coded as `"M"` and `"F"`.
+A good illustration is the raw data that is provided to you by a survey. A standard question for every questionnaire is the sex of the respondent. You remember from the previous question that this is a factor and when performing the questionnaire on the streets its levels are often coded as `"M"` and `"F"`.
```

Comment:
"A common question is the sex of the respondent, which is still often over-simplified to `"M"` and `"F"`. However many categories there are, this is a factor."

Reply:
Updated via https://github.com/datacamp/courses-intro-to-r/commit/a36c7926c7c68ba180be569535f23fbb893ac835